sycl/doc/extensions/proposed/sycl_ext_oneapi_device_image_backend_content.asciidoc

```diff
@@ -154,8 +152,7 @@ std::span<std::byte> ext_oneapi_get_content_backend_view() const;
 ----
 !====
 
-Available only when the compiler defines the `__cpp_lib_span` feature-test macro
-(which is defined in {cpp}20 and higher).
+Minimum C++ Version: {cpp}20
```

Comment:
Should this be in italics?
```suggestion
_Minimum C++ Version:_ {cpp}20
```

Reply:
Ah, I see.  My preference here would be to adopt a consistent style for all paragraphs rather than mixing the old and new styles.  I find the paragraph sub-headings really helpful when reading the specification, so something like a "_Deprecated_: SYCL 2020" looks better to me.

But I don't think we can solve this here, so I'll approve.  But maybe we should talk to some other folks to see if anybody else has a preference either way.